sw/device/lib/dif/dif_template.h

```diff
@@ -0,0 +1,253 @@
+// Copyright lowRISC contributors.
```

Comment:
Suggestion: This should probably have a different extension, like `.h.tpl` - a few bits of automation look for `dif_*.h` in this directory, and I don't want even more special cases there.

Reply:
It should also make clang-format not get confused, too.